libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -2515,6 +2515,22 @@ void AP_AHRS::writeExtNavVelData(const Vector3f &vel, float err, uint32_t timeSt
 #endif
 }
 
+// set the terrain SRTM altitude in meters above sea level
+// only used by optical flow when out of rangefinder range
+// Write terrain (derived from SRTM) altitude in meters above sea level
+void AP_AHRS::writeTerrainAMSL(float alt_amsl_m)
+{
+#if EK3_FEATURE_OPTFLOW_SRTM
```

Comment:
This should go on the outside of the method

Reply:
The reasons I didn't do this is it's inconsistent with the similiar writeOptFlowMeas method.  I think because the feature needs to be moved to AHRS instead of EKF3